dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/organisationunit/hibernate/HibernateOrganisationUnitStore.java

```diff
@@ -265,33 +264,44 @@ public Long getOrganisationUnitHierarchyMemberCount( OrganisationUnit parent, Ob
             Set<String> dataSetIds = SqlUtils.getArrayAsSet( rs, "ds_uid" );
 
             map.put( organisationUnitId, dataSetIds );
-        });
+        } );
 
         return map;
     }
 
     @Override
     public List<OrganisationUnit> getWithinCoordinateArea( double[] box )
     {
-        // can't use hibernate-spatial 'makeenvelope' function, because not available in
+        // can't use hibernate-spatial 'makeenvelope' function, because not
+        // available in
         // current hibernate version
         // see: https://hibernate.atlassian.net/browse/HHH-13083
 
         if ( box != null && box.length == 4 )
         {
             return getSession().createQuery(
-                    "from OrganisationUnit ou " + "where within(ou.geometry, " + doMakeEnvelopeSql( box ) + ") = true",
-                    OrganisationUnit.class ).getResultList();
+                "from OrganisationUnit ou " + "where within(ou.geometry, " + doMakeEnvelopeSql( box ) + ") = true",
+                OrganisationUnit.class ).getResultList();
         }
         return new ArrayList<>();
     }
 
     private String doMakeEnvelopeSql( double[] box )
     {
-        // equivalent to: postgis 'ST_MakeEnvelope' (https://postgis.net/docs/ST_MakeEnvelope.html)
+        // equivalent to: postgis 'ST_MakeEnvelope'
+        // (https://postgis.net/docs/ST_MakeEnvelope.html)
         return "ST_MakeEnvelope(" + box[1] + "," + box[0] + "," + box[3] + "," + box[2] + ", 4326)";
     }
 
+    @Override
+    public List<String> getUIDsCreatedBefore( Date date )

```

Comment:
This code should have been made generic and put in `HibernateIdentifiableObjectStore`.